orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -3746,6 +3746,8 @@ def forward(self, input1, bool_argument, int_argument, float_argument):
                 out = self.relu(out)
             return out
 
+    torch.manual_seed(0)
```

Comment:
let's control the atol and rtol instead of setting the manual seed. I think you can try with atol=1e-5, rtol=1e-4. That should do it.